src/drt/src/io/io_guide.cpp

```diff
@@ -855,7 +855,7 @@ void io::Parser::genGuides(frNet* net, vector<frRect>& rects)
   genGuides_pinEnclosure(net, rects);
 
   vector<map<frCoord, boost::icl::interval_set<frCoord>>> intvs(
-      tech->getLayers().size());
+      min((int)tech->getLayers().size(), TOP_ROUTING_LAYER+1));
```

Comment:
This doesn't ensure that no data will inserted above TOP_ROUTING_LAYER by genGuides_merge.  I suspect you may have introduced an out-bounds error.

Reply:
It ensures because I looked at the code that inserts into intvs. All insertion is bounded to < intvs.size()